roles/docker/tasks/main.yml

```diff
@@ -24,17 +24,29 @@
   action: "{{ ansible_pkg_mgr }} name=docker{{ '-' + docker_version if docker_version is defined and docker_version != '' else '' }} state=present"
   when: not openshift.common.is_atomic | bool and not docker_version_result | skipped and docker_version_result.stdout | default('0.0', True) | version_compare(docker_version, 'lt')
 
-# We're getting ready to start docker.  This is a workaround for cases where it
-# seems a package install/upgrade/downgrade has rebooted docker and crashed it.
+- name: Start the docker service
+  service:
+    name: docker
+    enabled: yes
+    state: started
+  register: start_result
+  ignore_errors: yes
```

Comment:
@detiber  Since scott it out for the rest of the day would we want to commit this now since it will unblock CI and then clean it up tomorrow?